ci-scripts/e2e-cypress.sh

```diff
@@ -85,5 +85,7 @@ else
     echo '-----'
     echo "Running Cypress end to end tests"
 
-    yarn e2e:run:ci"${SUITE}"
+    #yarn e2e:run:ci"${SUITE}"
+    yarn e2e:run:ci:core"${SUITE}"
+    #yarn e2e:run:ci:b2b"${SUITE}"
```

Comment:
How does the CI know it needs too run the full regression suite on `develop` and `epic` branch merges and run the core suite otherwise on feature branches?

Reply:
Further discussion needed with Gil about this subject.